DEFAULT_LICENCE.md

```diff
@@ -0,0 +1,20 @@
+/*
+ * This file is part of Betaflight.
```

Comment:
All license files should say part of betaflight and cleanflight as this project was forked from Cleanflight and all the target definitions were based on code from Cleanflight.

See other license files in the project for reference.

Reply:
No they shouldn't. This file is only a part of Betaflight.

Reply:
the content from the target configs (`target.h`) was derived from the `target.h` files from cleanflight.  all targets used one of the base targets from Cleanflight.

I refer you to the target.h file for the first target that was ever made by me, the OLIMEXINO.

https://github.com/cleanflight/cleanflight/blob/848a35ff408021566fcb616bb71b474d1fcf3bc4/src/main/target/OLIMEXINO/target.h

Reply:
I appreciate that any file that came as part of the fork should include reference to Cleanflight, and it will be my intention to continue that tradition. But from now on any new file in Betaflight will not refer to Cleanflight.